build.sbt

```diff
@@ -16,7 +16,7 @@
 lazy val scala211Version = "2.11.12"
 lazy val scala212Version = "2.12.10"
 lazy val sparkVersion = "2.4.0"
-lazy val grpcVersion = "1.30.2"
+lazy val grpcVersion = "1.35.0"
```

Comment:
please revert the grpc/netty/bigquery versions to those of version 0.19.1 ? When those were updated in 0.19.0 it caused problems to several users. We will update those after the v1 release

Reply:
Do you know exactly which library this was for?  I was hoping it was netty which I haven't touched here.  Do you have a repro of the issue to see if these changes are safe?

The bigquery/gax/grpc  updates are driven by needing the latest version of the proto files to support compression configuration.  Users can always use the serialized proto approach but I'd prefer to leave it undocumented.

Any thoughts on the other changes?

Reply:
(I tried just keeping the storage library and tests fail with this approach).

Reply:
grpc-netty has a dependency on netty

Reply:
reverted.